rook/main.py

```diff
@@ -45,28 +47,90 @@
 
 args = parser.parse_args()
 
-if args.load_average > 0 and hasattr(os, "getloadavg"):
-  #Note that this is also done before starting each test
-  while os.getloadavg()[0] > args.load_average:
-    print("Load Average too high, waiting ", os.getloadavg()[0])
-    time.sleep(1.0)
+if socket.gethostname() == "VDI-RavenW7-01":
```

Comment:
Hmmm.... dislike...please explain and change

Reply:
This is temporary.  Basically, as soon as the test script adds a '-l number' to the windows testing, this can be removed.  Alternatively, I can split part 9 in to a part that adds psutil, and a part that removes the deleting python, and remove this line.  We can commit part 9 psutil.  Then we change the windows testing script.  Then we can add the deleting python part.

Reply:
By the way, removing this is probably why #898 keeps failing the windows test.